script/e2e-test.sh

```diff
@@ -31,7 +31,10 @@ echo "IMAGE_REPO_SUFFIX: $IMG_MODIFIER"
 kubectl version
 
 # Install Tiller with TLS support
+kubectl -n kube-system create sa tiller
```

Comment:
we need to set the tiller service account as cluster-admin because by default, in a cluster created with `kind` the `default` service account in the `kube-system` namespace doesn't have permissions to read config maps in that namespace.

Reply:
Weird, I thought this was the case in GKE too - not sure how we were able to avoid it before?

Reply:
tbh I am not sure. My supposition is that depending on the user that creates the cluster, GKE default permissions fallback to those, so if an user with privileges creates a cluster (like the account set here) it's not necessary to create the extra service account. I have tried with my personal account and for me it requires to use the tiller serviceaccount.

Reply:
Hmm that's pretty strange, so the service account here has admin privileges in the project?

Reply:
yes, it has more privileges than a normal account